apps/gitness/src/App.tsx

```diff
@@ -57,6 +59,24 @@ export default function App() {
         {
           path: 'executions',
           element: <ExecutionsPage />
+        },
+        {
+          path: ':spaceId/:repoId',
+          element: <RepoLayout />,
+          children: [
+            {
+              index: true,
+              element: <>Repos list</>
```

Comment:
repo list page is already implemented..
but why is it a child of `spaceId/repoId`? that should be the repo details page